.github/workflows/release.yml

```diff
@@ -367,19 +423,32 @@ jobs:
     - name: Create download folder
       run: |
         mkdir -p ${{ matrix.os }}-${{ matrix.arch }}-bin
+        mkdir -p ubuntu-tracing-bin
 
     - name: Download pre-built collator binary
       uses: actions/download-artifact@v3
       with:
         name: astar-${{ matrix.os }}-latest-${{ matrix.arch }}
         path: ${{ matrix.os }}-${{ matrix.arch }}-bin
 
+    - name: Download pre-built collator binary for evm tracing
+      uses: actions/download-artifact@v3
+      with:
+        name: astar-ubuntu-tracing-latest-x86_64-unknown-linux-gnu
```

Comment:
Name should be 
astar-evm-tracing-ubuntu-latest-x86_64-unknown-linux-gnu

Because, 
- ubuntu version is the latest, not tracing.
- evm should be added, because 1) others don't know what is `tracing` without any context, 2) we might have something other tracing.

Reply:
thanks for fix in #928